paramgen_dest.go

```diff
@@ -153,26 +138,50 @@ func (DestinationConfig) Parameters() map[string]sdk.Parameter {
 			Type:        sdk.ParameterTypeBool,
 			Validations: []sdk.Validation{},
 		},
+		"queue.name": {
```

Comment:
Btw, why don't we put the `queue.*` configuration options in the shared `Config`? These config values seem to be present both in the source and destination.